client/signup/jetpack-connect/install-step.jsx

```diff
@@ -7,20 +7,42 @@ import React from 'react';
  * Internal dependencies
  */
 import Card from 'components/card';
+import { preventWidows } from 'lib/formatting';
 
 export default React.createClass( {
 	displayName: 'JetpackInstallStep',
 
+	renderText() {
+		const { action } = this.props;
+		let { text } = this.props;
+
+		if ( ! action ) {
+			return (
+				<span>
+					{ preventWidows( text ) }
+				</span>
+			);
+		}
+
+		return (
+			<span>
+				{ text }
+				<span> </span>
```

Comment:
Nah, it's not necessary to run it on both. I only meant that regardless of the markup, that text is read as a paragraph. The intent of preventWidows is to keep the last word of a block of text from landing on its own line. Since the text is all 'inline' it would never be possible to have a widow on the text leading up to the 'action' which in this case is simply another sentence. The only reason to run preventWidows on this particular paragraph would be to prevent the last word of the link from landing on its own line—perhaps we can just run the method on the link and skip some of the complication?